2.0/m_rehashsslsignal.cpp

```diff
@@ -50,6 +50,9 @@ class ModuleRehashSSLSignal : public Module
 		if (!signaled)
 			return;
 
+		std::string m = "SSL credentials have been reloaded via terminal on " + ServerInstance->Config->ServerName ;
+		ServerInstance->SNO->WriteGlobalSno('a', m);
+		
 		ServerInstance->Logs->Log("m_rehashsslsignal", DEFAULT, "Got SIGUSR1, reloading SSL credentials");
```

Comment:
Could probably use this existing message for both Log and SNOTICE. No need to say what server in the message as the SNOTICE will come from that server.